tests/SMS77Test.php

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
I appreciate you adding a test for the api calls, but please delete the file.  I am not a fan of testing API points because there is no good way of storing the credentials to allow the tests to run.  These tests are currently making the pull request fail the tests.